source/mobile/mobile-appconfig.rst

```diff
@@ -50,3 +50,22 @@ As part of AppConfig, EMM administrators can set the following additional config
 4. **Remotely Wipe App:** Use the EMM tool to distribute the app to devices as a managed application so it can be remotely wiped. If the app was previously installed, mark it so the EMM converts the app to a managed app.
 
 Other configurations may be available depending on your EMM provider.
+
+Adding Mattermost mobile apps to your EMM provider
+------------------------------------------------------------------------
+
+Because the Mattermost mobile apps have support for AppConfig, you can deploy the apps found in Google Play and the Apple App Store, or you can choose to deploy them as in-house apps.
+
+AppConfig EMM Members
+------------------------
+
+Mattermost mobile apps can be deployed with any of the EMM providers listed in the AppConfig Website.
+
+Mattermost Support for AppConfig in Native Mobile apps
+---------------------------------------------------------------------
+
+What is the AppConfig Community?
```

Comment:
We already have `What is AppConfig?` section at the top. Should we combine the two? Or is this section for more information?

Reply:
Not sure, I thought they were two different topics. Will check with Elias.